test/test_wordcloud_cli.py

```diff
@@ -149,7 +150,26 @@ def test_cli_writes_image(tmpdir, tmp_text_file):
     args, text, image_file = cli.parse_args(['--text', str(tmp_text_file), '--imagefile', str(tmp_image_file)])
     cli.main(args, text, image_file)
 
-    # expecting image to be written
+    # expecting image to be written to imagefile
+    assert tmp_image_file.size() > 0
+
+
+# capsysbinary should be used here, but it's not supported in python 2.
+def test_cli_writes_to_stdout(tmpdir, tmp_text_file):
+    # ensure writing works with all python versions
+    tmp_image_file = tmpdir.join("word_cloud.png")
+
+    tmp_text_file.write(b'some text')
+
+    originalBuffer = sys.stdout.buffer
+    sys.stdout.buffer = tmp_image_file.open('wb+')
```

Comment:
This actually breaks with python38 and newer because `sys.stdout.buffer` is now read-only.